chef/cookbooks/metasploitable/recipes/cleaner.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Cookbook:: metasploitable
+# Recipe:: cleaner
+#
+# Copyright:: 2017, The Authors, All Rights Reserved.
+
+directory 'C:\vagrant' do
+  recursive true
+  action :delete
```

Comment:
Are you trying to remove a shared folder?

Reply:
Yes. If the folder `C:\vagrant` exists in the packer build, some issues occur when starting the box using vagrant.

Reply:
Hmm ok, I think this can be dangerous if user mounted a local directory to the guest at `C:\vagrant`.. no?